test1.js

```diff
@@ -3,3 +3,10 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+
+var foo77 = '' ;
+
+if(foo77  === "baaaaaaaaaar") {
+  console.log( 'heaaaaaaay' )
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26923)
```suggestion
  console.log( "heaaaaaaay" )
```